stylesheets/components/button.css

```diff
@@ -0,0 +1,104 @@
+.btn {
```

Comment:
Add following property for all the buttons.
.btn:focus{outline:none}
When we click on the button in mobile phone, a default outline wil be visible. With above property, it will not happen.

Reply:
Added in the latest commit

Reply:
Are bro aise nhi.
.btn:focus{outline: none}